### PR TITLE
Create workflow graph

### PR DESCRIPTION
**Pull request by mtitov**
Enables workflow graph creation (later will be expanded towards provenance graph)

### CODE REVIEW

File: src/radical/entk/utils/provenance.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        for stage in pipelines.stages:
+            graph[pipeline.uid][stage.uid] = []
+
+            for task in stage.tasks:
+                graph[pipeline.uid][stage.uid].append(task.uid)
+
+            graph[pipeline.uid][stage.uid].sort()
+
+    if output_file:
+        if '/' not in output_file:
```
Comment (andre-merzky) [inline src/radical/entk/utils/provenance.py:36]: Why do you want to enforce a `/` in the file name?

File: src/radical/entk/utils/provenance.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+            setdefault(pipeline_uid, {}).\
+            setdefault(stage_uid, []).\
+            append(task_uid)
+
+    for pipeline_uid in graph:
+        for stage_uid in graph[pipeline_uid]:
+            graph[pipeline_uid][stage_uid].sort()
+
+    if output_file:
+        if '/' not in output_file:
```
Comment (andre-merzky) [inline src/radical/entk/utils/provenance.py:72]: same

**Review by andre-merzky — APPROVED** [review]
LGTM

File: src/radical/entk/utils/provenance.py (hunk 1 above)
Comment (mtitov) [reply src/radical/entk/utils/provenance.py:36]: thought to be explicit where file is saved. Should I remove this check? (and only to keep `.json` check)

Comment (andre-merzky) [reply src/radical/entk/utils/provenance.py:36]: Yeah, I would suggest to remove.  Thanks!

**Pull request merged**